src/browser/ui/ReactDOMComponent.js

```diff
@@ -151,6 +154,18 @@ function validateDangerousTag(tag) {
   }
 }
 
+function validateDangerousAttributeName(attributeName) {
+  if (!hasOwnProperty.call(validatedAttributeNameCache, attributeName)) {
+    invariant(
+      VALID_ATTRIBUTENAME_REGEX.test(attributeName),
+      'Invalid attribute name: `%s`',
+      attributeName
+    );
+    validatedAttributeNameCache[attributeName] = true;
+  }
+  return attributeName;
```

Comment:
I understand the style but if we don't use it like that, there's no need to do it.